src/components/main/main.jsx

```diff
@@ -1,8 +1,22 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
+const renderFilmsCard = (films) => {
```

Comment:
исходя из семантики языка так будет точнее:
```suggestion
const renderFilmCards = (films) => {
```